queries/appServices/stopped-appServices/query.txt

```diff
@@ -0,0 +1 @@
+extend state = aliases['Microsoft.Web/sites/state'] | where type =~ 'Microsoft.Web/Sites' and state =~ 'Stopped' | project name, kind, state
```

Comment:
Hi @whaakman,

Nice scenario. Can you just break each piped statement on a new line please? It makes it easier to read and the CI script will auto remove them when you run it.

Reply:
Done :)